product_landing_page/index.html

```diff
@@ -80,6 +80,14 @@ <h2 class="advantages-section__title">Quality Assurance</h2>
 
     </section>
 
+    <div class="container__item">
```

Comment:
W tym przypadku nie potrzebujemy diva `container__item` wrappującego `iframe`, bo jest to jego jedyne dziecko. Także śmiało można go usunąć - wycentrowaniem `iframe`a zajmie się `container` :)